packages/x-grid-data-generator/src/useDemoData.ts

```diff
@@ -37,13 +37,15 @@ function extrapolateSeed(rowLength, columns, data) {
       const index = Math.round(Math.random() * (seed.length - 1));
 
       if (column.field === 'id') {
+        // @ts-ignore
         row[column.field] = `id-${i + seed.length}`;
       } else {
+        // @ts-ignore
```

Comment:
I don't understand why we need to add ts ignore here

Reply:
Well spotted, I missed the revert on this fi'e.

Reply:
you cannot put  // @ts-ignore every time there is an error that you don't understand.
You're probably missing the row type or a type in rows or something like that.

Reply:
stop adding //ts ignore as a practice please

Reply:
Damien, the initial objective was to enforce the `noImplictAny` rule so that the CI would fail. It's was meant to put a constraint in place to enforce better typing for code we will write, at the expense of adding any/ignore for the code that was written in the past and that is, right now untyped. We agreed on going with the soften approach: try to spot the cases in during the review workflow and to revert the changes here. It's one case where I forgot to revert.